### PR TITLE
Added Glance.app v1.1

### PR DESCRIPTION
**Pull request by crunchtime-ali**
Adds [Glance](https://github.com/samuelmeuli/glance) which is an All-In-One Quick Look plugin that uses the new Quick Look API. Since a lot of the Quick Look plugins use the outdated API and are not sandboxed `Glance` is gaining quite some traction.

The caveat that you have to start the app once manually to set up the Quick look plugin is a request by the author:

@samuelmeuli writes in https://github.com/samuelmeuli/glance/issues/11#issuecomment-630335561 
> I think the user should open the app by themselves. Automatically executing a binary that was downloaded from the internet seems a bit problematic to me. And the user might want to verify that the app is sandboxed before opening it. Maybe a note in caveats is enough?

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/glance.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  appcast 'https://github.com/samuelmeuli/glance/releases.atom'
+  name 'Glance'
+  homepage 'https://github.com/samuelmeuli/glance'
+
+  depends_on macos: '>= :catalina'
+
+  app 'Glance.app'
+
+  zap trash: [
+               '~/Library/Application\ Scripts/com.samuelmeuli.Glance/',
```
Comment (vitorgalvao) [inline Casks/glance.rb:15]: Remove the backslashes. They’re inside quotes, meaning they’re interpreted literally.

Comment (vitorgalvao) [reply Casks/glance.rb:15]: Also, remove the trailing slashes.

File: Casks/glance.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+               '~/Library/Application\ Scripts/com.samuelmeuli.Glance.QLPlugin/',
+               '~/Library/Containers/com.samuelmeuli.Glance/',
+               '~/Library/Containers/com.samuelmeuli.Glance.QLPlugin/',
+               '~/Library/Group\ Containers/group.com.samuelmeuli.glance/',
+             ]
+
+  caveats <<~EOS
+    You must start #{appdir}/Glance.app once manually to setup the QuickLook plugin.
```
Comment (vitorgalvao) [inline Casks/glance.rb:23]: Where is the quicklook plugin after one opens the app? It should be removed on `uninstall`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/glance.rb (hunk 1 above)
Comment (crunchtime-ali) [reply Casks/glance.rb]: Done @vitorgalvao

File: Casks/glance.rb (hunk 2 above)
Comment (crunchtime-ali) [reply Casks/glance.rb:23]: As I understand it the Quick Look plugin resides in the app bundle at: /Applications/Glance.app/Contents/PlugIns/QLPlugin.appex

Comment (vitorgalvao) [reply Casks/glance.rb:23]: And does it stay there after it’s registered by the app?

**Comment by vitorgalvao** [comment]
@samuelmeuli Can’t there be a way to install the quicklook plugin without users having to manually install the app?

Homebrew Cask users expect to not have to do manual tasks to finish the installation after installing a cask. I’ve read [your concerns](https://github.com/samuelmeuli/glance/issues/11#issuecomment-630335561), but they don’t apply—we already install `pkg`s without interaction, so this would be no different.

File: Casks/glance.rb (hunk 2 above)
Comment (samuelmeuli) [reply Casks/glance.rb:23]: Yes, removing the app should be enough.

**Comment by samuelmeuli** [comment]
> Can’t there be a way to install the quicklook plugin without users having to manually install the app?

I don't think so. The container app is required [according to this Stack Overflow answer](https://stackoverflow.com/a/39628355/6767508).

> Homebrew Cask users expect to not have to do manual tasks to finish the installation after installing a cask. I’ve read your concerns, but they don’t apply—we already install pkgs without interaction, so this would be no different.

Fair enough :)

File: Casks/glance.rb (hunk 2 above)
Comment (crunchtime-ali) [reply Casks/glance.rb:23]: @vitorgalvao Yes, it does.

**Comment by vitorgalvao** [comment]
Thank you for the answers.

**Pull request merged**